src/applications/edu-benefits/10203/sass/10203_stem.scss

```diff
@@ -9,7 +9,55 @@
 @import "~@department-of-veterans-affairs/formation/sass/modules/m-omb-info";
 @import "../../../../platform/forms/sass/m-form-confirmation";
 @import "~@department-of-veterans-affairs/formation/sass/modules/va-pagination";
+@import "~uswds/src/stylesheets/core/utilities";
 
+.process-1 {
```

Comment:
Can this have a better name?

how much of this is different from the `process` class that was previously used?

Reply:
The original process would override the  the circle bullet point making them square, I had to pull it out and so we can add the circle bullets.